include/engine/phantom_node.hpp

```diff
@@ -177,7 +186,7 @@ struct PhantomNode
 };
 
 #ifndef _MSC_VER
-static_assert(sizeof(PhantomNode) == 52, "PhantomNode has more padding then expected");
+static_assert(sizeof(PhantomNode) == 60, "PhantomNode has more padding then expected");
```

Comment:
@BergWerkGIS I cannot verify on my system whether this size captures the size on a Windows system correctly and the AppVeiyor builds are currently not working. Could you maybe verify whether this `static assert` works?